<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="display_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>481</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_video">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>180</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetFiles">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>비디오 선택</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>430</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>  시작</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>600</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_max">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>350</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_min">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>380</y>
     <width>231</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="display_binary">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>360</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>340</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_minThr">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>370</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>340</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>thr_max</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_maxThr_3">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>370</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>thr_min</string>
   </property>
  </widget>
  <widget class="QLabel" name="display_graph">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>641</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_videoTime">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>580</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00:00 / 00:00:00</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showGraph">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>560</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Graph</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_saveDirectory">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>csv 저장 폴더 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_saveDirectory">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>50</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
